### DISCUSSION TITLE
Some reference source code from other teams last year

### DISCUSSION THREAD

**Discussion by coverbeek** [frc5567/2023-Robot#98] [Team Posts]
One of Danny's friends pointed us at their source code from last year for looking at their vision processing code. I am sharing it as this could be something some of you might be interested in checking out.
https://github.com/FRC2832/Robot_2017

Another team that used PixyCam last year has their source here:
https://github.com/trc492/Frc2017FirstSteamWorks

And one more site that I found some interesting information on:
https://github.com/GirlsOfSteelRobotics/Docs/wiki/Camera-and-Vision-Processing

**Comment by coverbeek** [comment]
Relevant to some of the discussion tonight at practice: https://github.com/WPIRoboticsProjects/GRIP

**Comment by JooshO** [comment]
This is code used for connecting the rio with the Arduino and Pixy for vision 
https://github.com/FRC5188/ArduinoPixyAndRoboRIO
